### PR TITLE
fix broken mlflow model build

### PR DESCRIPTION
**Pull request by RafalSkolasinski**
Fix broken mlflow model build.

Pining of protobuf required due to https://github.com/mlflow/mlflow/issues/5949

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **rafalskolasinski**
You can assign the PR to them by writing `/assign @rafalskolasinski` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: servers/mlflowserver/models/elasticnet_wine/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 mlflow==1.19.0
 conda-pack==0.6.0
+protobuf<4.0
```
Comment (RafalSkolasinski) [inline servers/mlflowserver/models/elasticnet_wine/requirements.txt:3]:
```suggestion
# pinning protobuf due to https://github.com/mlflow/mlflow/issues/5949
protobuf<4.0
```

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[axsaucedo](https://github.com/SeldonIO/seldon-core/pull/4155#issuecomment-1157501261 "Approved")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**